tests/ui_tests/test_cluster_list_page.py

```diff
@@ -780,6 +784,84 @@ def test_open_by_tab_cluster_status_page(self, app_fs, create_community_cluster)
+        one_negative = 'successful 0/1'
+
+        success_status = [
+            StatusRowInfo(
+                icon=True, group_name=CLUSTER_NAME, state='successful 2/2', state_color=SUCCESS_COLOR, link=None
+            ),
+            StatusRowInfo(icon=False, group_name='Hosts', state=one_successful, state_color=SUCCESS_COLOR, link=None),
+            StatusRowInfo(icon=True, group_name=None, state=None, state_color=None, link='test-host'),
+            StatusRowInfo(
+                icon=False, group_name='Services', state=one_successful, state_color=SUCCESS_COLOR, link=None
+            ),
+            StatusRowInfo(
+                icon=True, group_name=SERVICE_NAME, state=one_successful, state_color=SUCCESS_COLOR, link=None
+            ),
+            StatusRowInfo(icon=True, group_name='first', state=one_successful, state_color=SUCCESS_COLOR, link=None),
+            StatusRowInfo(icon=True, group_name=None, state=None, state_color=None, link='test-host'),
+        ]
+        host_negative_status = [
+            StatusRowInfo(
+                icon=True, group_name=CLUSTER_NAME, state='successful 1/2', state_color=NEGATIVE_COLOR, link=None
+            ),
+            StatusRowInfo(
+                icon=False, group_name='Hosts', state=one_negative, state_color=NEGATIVE_COLOR, link=None
+            ),
+            StatusRowInfo(icon=True, group_name=None, state=None, state_color=None, link='test-host'),
+            StatusRowInfo(
+                icon=False, group_name='Services', state=one_successful, state_color=SUCCESS_COLOR, link=None
+            ),
+            StatusRowInfo(
+                icon=True, group_name=SERVICE_NAME, state=one_successful, state_color=SUCCESS_COLOR, link=None
+            ),
+            StatusRowInfo(icon=True, group_name='first', state=one_successful, state_color=SUCCESS_COLOR, link=None),
+            StatusRowInfo(icon=True, group_name=None, state=None, state_color=None, link='test-host'),
+        ]
+        host_and_component_negative_status = [
+            StatusRowInfo(
+                icon=True, group_name=CLUSTER_NAME, state='successful 0/2', state_color=NEGATIVE_COLOR, link=None
+            ),
+            StatusRowInfo(
+                icon=False, group_name='Hosts', state=one_negative, state_color=NEGATIVE_COLOR, link=None
+            ),
+            StatusRowInfo(icon=True, group_name=None, state=None, state_color=None, link='test-host'),
+            StatusRowInfo(
+                icon=False, group_name='Services', state=one_negative, state_color=NEGATIVE_COLOR, link=None
+            ),
+            StatusRowInfo(
+                icon=True, group_name=SERVICE_NAME, state=one_negative, state_color=NEGATIVE_COLOR, link=None
+            ),
+            StatusRowInfo(icon=True, group_name='first', state=one_negative, state_color=NEGATIVE_COLOR, link=None),
+            StatusRowInfo(icon=True, group_name=None, state=None, state_color=None, link='test-host'),
+        ]
```

Comment:
What do you think about list comprehension here? Or it's that big too

Reply:
changed